### PR TITLE
Problem: (cro-1335) no guide for cbindings android compile on mac #1335

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: add document

### CODE REVIEW

File: cro-clib/android.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ export NDK_HOME=$ANDROID_HOME/ndk/21.0.6113669
...
 ```
 
+## check crate-type
+1. vi Cargo.toml  in cro-clib folder
+2. modify crate-type as staticlib only  
+```
+[lib]
+name = "cro_clib"
+crate-type =["staticlib"]
+```
```
Comment (tomtau) [inline cro-clib/android.md:28]: Can this change be made directly in Cargo.toml? Most platforms need `staticlib`, so I guess it'll make sense to make it a default crate-type, so it's not necessary for someone to modify Cargo.toml (if someone really needs `cdylib` for some reason, they can tweak it... but for 99% cases, `staticlib` should be fine?)

Comment (tomtau) [reply cro-clib/android.md:28]: iOS also needs staticlib?

File: cro-clib/android_mac.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+
+## check crate-type
+1. vi Cargo.toml  in cro-clib folder
+2. modify crate-type as staticlib only  
+```
+[lib]
+name = "cro_clib"
+crate-type =["staticlib"]
+```
```
Comment (tomtau) [inline cro-clib/android_mac.md:50]: can it be without this step if Cargo.toml is changed?

File: cro-clib/android.md (hunk 1 above)
Comment (leejw51crypto) [reply cro-clib/android.md:28]: yes, both staticlib
in ios, all libs are static

i'll modify

staticlib is find & stable

File: cro-clib/android_mac.md (hunk 2 above)
Comment (leejw51crypto) [reply cro-clib/android_mac.md:50]: i'll remove it

**Pull request merged**